### DISCUSSION TITLE
Migration to Scala Steward GitHub Action

### DISCUSSION THREAD

**Discussion by ptrdom** [playframework/playframework#11555] [Ideas]
Since https://github.com/scala-steward-org/repos no longer accepts adding new repos, I wonder if we could setup https://github.com/scala-steward-org/scala-steward-action for use with orgs repos. This needs a bit of setup, because the best practice is to setup a GitHub App to perform Scala Steward's actions, I assume that we could have one of those for the whole org. 

The main reason why I am asking is that I want Scala Steward to be able to execute scala-cli script in https://github.com/playframework/play-scala-react-seed for PRs it produces, and with it being a part of workflows, I could just install the necessary software for it to be capable of doing so.

**Comment by ihostage** [comment]
Hi, @ptrdom! Yes, we already thought to migrate Scala Steward activity from `scala-steward-org/repos` to one of our organization repo (for example `.github`). And thank you for adding yet one reason for do that in the future :+1:

**Comment by mkurz** [comment]
> Since https://github.com/scala-steward-org/repos no longer accepts adding new repos

You still can add new repos. I think they just moved to a different server. See https://github.com/scala-steward-org/scala-steward#quick-start-guide
You just need to open a pr against https://github.com/VirtusLab/scala-steward-repos/blob/main/repos-github.md that's it.

**Reply by ptrdom** [reply]
Oh neat, so that is why it is still working for existing repos 😄

**Reply by mkurz** [reply]
Yeah, all good.
